packages/ng-table-builder/src/table-builder.component.ts

```diff
@@ -381,7 +381,7 @@ export class TableBuilderComponent<T>
     }
 
     public filterBySubstring(substring: Nullable<string>): void {
```

Comment:
`filterBySubstring(substring: Nullable<Any>)`

Reply:
I knew you would write it)
I think we also need to add a check
substring?.toString === Object.prototype.toString ? substring.toString() : null

Reply:
fixed